Makefile

```diff
@@ -171,6 +172,19 @@ else
 KUSTOMIZE=$(shell which kustomize)
 endif
 
+# download etcd if necessary
+# requires root privileges
+etcd:
```

Comment:
do we need this?

Reply:
Only initially, but yes. `etcd`, along with other `KUBEBUILDER_ASSETS` are required for `kubebuilder` (`operator-sdk` dependency) to work. Specifically, it's required by the `env-test` binary to run e2e tests: https://book.kubebuilder.io/reference/envtest.html

Reply:
To set this up initially, contributors only need to do `make etcd` which would install all required `kubebuilder` binaries, including `etcd`.